bin/rake

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env ruby
```

Comment:
What's the current deal with binstubs? Is this the standard in ruby land at this point vs `bundle exec rake`?

Reply:
I suppose it's somewhere between a partial standard and merely a convenience.  But `bundle gem $gemname` only creates `bin/setup` and `bin/console`.

I _personally_ find it slightly nicer to use the binstubs, so I like to create them for rake and rspec and anything else I'm running frequently.  I don't know how widespread or "standard" that is though, but rails does it, bundler makes it easy to do, and I do sometimes see it in other projects.

Reply:
I'm cool with binstubs, that's fine by me.